src/static-landing/index.html

```diff
@@ -18,68 +18,93 @@
         </div>
       </div>
       <div class="container">
-        <a class="navbar-brand" title="BC Parks Beta" href="#">
+        <a class="navbar-brand d-none d-md-inline" title="BC Parks Beta" href="#">
           <img src="images/bcparks_primary_reversed.svg" height="100" alt="BC Government Logo" />
         </a>
+        <a class="navbar-brand d-inline d-md-none" title="BC Parks Beta" href="#">
+          <img src="images/bcparks_wordmark_white.svg" height="60" alt="BC Parks Logo" />
+        </a>        
       </div>
     </nav>
     <div class="container-fluid beta-welcome">
       <h1 class="text-center beta-welcome_main-heading">BCPARKS.CA BETA</h1>
-      <p class="text-center beta-welcome_sub-heading">A new experience is coming this year!</p>
+      <p class="text-center beta-welcome_sub-heading">
+        You asked. We listened. <br />A new digital experience of BC Parks is coming this year.
+      </p>
       <div class="text-center beta-welcome_screenshots-container">
         <img class="beta-welcome_screenshots" src="images/laptop-mobile-demo.png" alt="Beta Screenshots" />
       </div>
     </div>
-    <div class="container">
-      <p class="text-center mx-auto beta-description">
-        BC Parks is on a mission to improve our digital services. Through extensive research and collaboration across
-        government ministries, our 2022 experience will help park visitors, park operators and BC Parks staff to better
-        plan for, manage and visit BC Parks. This is just the beginning – and we invite you to engage with us along the
-        way!
+    <div class="container beta-description">
+      <h2>Everyone loves being in British Columbia’s parks.</h2>
+      <p>
+        In 2022, BC Parks is on a mission to ensure your experience of parks – from trip planning, to booking a
+        campsite, to contributing to conservation – is just as enjoyable, seamless, and impactful no matter whether
+        you’re in a park, on your laptop, or on the road in between. To meet this goal, we’re setting out to
+        significantly and continuously improve the services we provide to BC’s parks and the people who visit and value
+        them.
+      </p>
+      <p>
+        Through extensive research with park visitors, and in collaboration with the Exchange Lab (the BC Government’s
+        incubator and accelerator for digital innovations like the Healt h Gateway) and some of #BCTech’s leading
```

Comment:
*Health

Reply:
Fixed in https://github.com/bcgov/bcparks.ca/pull/249/commits/400e913265603d214b0214e25f0528652893be48